Playground for the new commondata format.ipynb

```diff
@@ -483,20 +505,198 @@
     "data_df"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 61,
+   "id": "41394702",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dataclasses import dataclass\n",
+    "from pathlib import Path\n",
+    "import typing\n",
+    "\n",
+    "from validobj.custom import Parser\n",
+    "from validobj import ValidationError, parse_input\n",
+    "\n",
+    "\n",
+    "from validphys import convolution\n",
+    "\n",
+    "\n",
+    "# Scalar parsers\n",
+    "@Parser\n",
+    "def ValidPath(path_str: str) -> Path:\n",
```

Comment:
Do we want things with more checks, like ExistingFilePath? For fun I tried to reach the error using the visible keys on my keyboard and could not.

Reply:
I'll see when I actually use it but I'm not sure at this stage the path will be testable for existence

Reply:
Should be fine either way. Don't have to check for everything at this stage but it sure is nice when sensible.